src/index/create.rs

```diff
@@ -229,6 +229,20 @@ impl IndexCreateStatement {
         &self.index
     }
 
+    pub(crate) fn get_name(&self) -> String {
+        if let Some(name) = self.index.name.clone() {
+            return name;
+        }
+        let prefix = if self.is_primary_key() {
+            "pri"
```

Comment:
I'm not sure it's a good solution to hard-code prefixes in the library... @billy1624 what do you think?

Reply:
I think it is ok in these cases because:
it was like that before at other locations you can already find these prefixes.
This is merely a fallback if nothing has been specified by the user.

Reply:
CC @billy1624 @tyt2y3)